paasta_tools/paasta_metastatus.py

```diff
@@ -345,13 +447,49 @@ def print_output(argv: Optional[List[str]]=None) -> None:
             for line in format_table(all_rows):
                 print_with_indent(line, 4)
     metastatus_lib.print_results_for_healthchecks(marathon_summary, marathon_ok, marathon_results, args.verbose)
+    metastatus_lib.print_results_for_healthchecks(kube_summary, kube_ok, kube_results, args.verbose)
+    if args.verbose > 1 and kube_enabled:
```

Comment:
I hate how big this function is already :-(

But really don't feel obliged to tackle that here